### PR TITLE
added kendo UI missing fields

### PR DESCRIPTION
**Pull request by gayan85**
case 1. Add a new type definition.
- [ ] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [ ] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

**Comment by mhegazy** [comment]
this looks like a big change, @telerik can you review?

**Comment by mhegazy** [comment]
@telerik 🔔

**Comment by mhegazy** [comment]
@dawidrylko, @chrisdrobison, @clabough, @tsvetomir, and @mmckenziedev can you review this change please.

**Comment by dawidrylko** [comment]
@telerik should review pull request. There are too many changes.

:alarm_clock:

**Comment by tsvetomir** [comment]
I believe that some kind of auto-format was applied in c8d5f64.

@gayan85 Please, force push only the actual changes in your branch so we can evaluate them.

**Comment by mmckenziedev** [comment]
I agree that this request has too many changes. I recommend having one change request that contains only the actual changes, and submit the formatting changes in a separate request. Combining the two just makes it awkward to evaluate.

**Comment by paulvanbrenk** [comment]
Closing this, as the actual changes are hidden by the formatting changes. Please create a separate PR for each.

**Comment by paulvanbrenk** [comment]
@mmckenziedev looks like the PR is a lot cleaner, can you take another look?

File: kendo-ui/kendo-ui.d.ts [hunk 1]
```diff
@@ -500,15 +503,15 @@ declare namespace kendo.data {
...
+        get(): void;
     }
 
     class TemplateBinding extends Binding {
         constructor(source: kendo.Observable, path: string, template: Function);
         render(value: Object): string;
     }
 
-    namespace binders { }
+    module binders { }
```
Comment (paulvanbrenk) [inline kendo-ui/kendo-ui.d.ts]: `namespace` is prefered

Comment (gayan85) [inline kendo-ui/kendo-ui.d.ts]: ok sure

**Comment by mmckenziedev** [comment]
:thumbsup:

File: kendo-ui/kendo-ui.d.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 /// <reference path="../jquery/jquery.d.ts" />
 
+
```
Comment (dawidrylko) [inline kendo-ui/kendo-ui.d.ts]: Please delete this empty line.

**Comment by dawidrylko** [comment]
:+1:

**Comment by paulvanbrenk** [comment]
@gayan85 if you can make the final two changes I'll merge this in.

**Comment by gayan85** [comment]
@paulvanbrenk i have updated. thanks

**Pull request merged**